lightning/src/routing/gossip.rs

```diff
@@ -1796,6 +1796,12 @@ impl ReadOnlyNetworkGraph<'_> {
 		self.channels.get(&short_channel_id)
 	}
 
+	#[cfg(c_bindings)] // Non-bindings users should use `channels`
```

Comment:
Maybe start the comment here and below on a fresh line? Also, isn't this first line kinda redundant since non-bindings users actually have no other choice than using the `channel`/`nodes` methods?

Reply:
Ah, I see! Still not sure about telling a group of users that another group should use something else, but really doesn't matter that much.